lib/lotus/model.rb

```diff
@@ -27,6 +27,16 @@ class NonPersistedEntityError < ::StandardError
     class InvalidMappingError < ::StandardError
     end
 
+    # Error for invalid query
+    # It's raised when a query is malformed
+    #
+    # @since 0.3.1
+    class InvalidQueryError < ::StandardError
+      def initialize(message = nil)
```

Comment:
it seems to me there is no case where message is nil, should we remove the default message?